aws/src/integration/java/org/apache/iceberg/aws/s3/TestS3FileIO.java

```diff
@@ -966,6 +969,138 @@ public void noStorageCredentialConfiguredWithoutCredentialsInProperties() {
         .hasMessageContaining("Unable to load credentials from any of the providers");
   }
 
+  @Test
+  public void testVectoredRead() throws Exception {
```

Comment:
Let's remove these tests from this PR and include them in the next PR that implements the S3 vectored read.